server/db/recreate-schema.sql

```diff
@@ -1,9 +1,32 @@
--- Drop tables in case they already exist
+DROP TABLE if exists documents;
+DROP TABLE if exists languages;
 DROP TABLE if exists users;
 
--- Create tables
+create type user_role as enum ('Translator', 'User');
```

Comment:
Can we use Interpreter instead of Translator as we started to use the term Interpreter everywhere else.

Reply:
yes, I think it is a good idea to keep interpreter